source/manual/common-aws-tasks-for-2nd-line-support.html.md

```diff
@@ -247,18 +121,18 @@ decisions have been made.
 
 ## How are instances provisioned?
 
-We typically use Terraform config in [govuk-aws][] to provision infrastructure,
+We typically use Terraform config in [govuk-aws] to provision infrastructure,
```

Comment:
Should `such as EC2 instances` be removed here?

Reply:
Good question — we could probably just delete this whole section at this point, but for now I've reworded it to make it a bit clearer to the reader that we're talking about legacy infrastructure in this section.